src/util/framerate.ts

```diff
@@ -183,8 +183,11 @@ export class DownsamplingBasedOnFrameRateCalculator {
   constructor(
     public numberOfStoredFrameDeltas: number = 10,
     private maxDownsamplingFactor: number = 8,
-    private desiredFrameTimingMs = 1000 / 60,
+    private desiredFrameTimingMs = 1000 / 30,
     private downsamplingPersistenceDurationInFrames = 15,
+    /** If true, logs factor changes and timing stats to console */
+    private enableLogging: boolean = false,
+    /** If true, always use MIN timing regardless of supplied method argument */
```

Comment:
Was this comment related to a param that was added and then removed?

Reply:
indeed! I was thinking of exploring MIN as the parameter because i was wanting to be more optimistic, but I decided that was extra complicated and probably using MEDIAN was sufficient if we wanted to be more optimistic than MEAN, but really after i realized that the target was being set to 8ms and that was the far more important adjustment to make.